v2/models/coredata/event.go

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package coredata
```

Comment:
Same comment w/r/t inclusion of service name

Reply:
Updated as suggested.